DOCS/xml-fields.md

```diff
@@ -0,0 +1,177 @@
+  * Note that `<standardLicenseHeader>` should _only_ be used if it is defined by the license text itself!
+  * Typically, this would be located in an appendix titled something like "How to apply the license to your work"
+
+### Paragraphs and newlines
+
+Separate paragraphs should be surrounded by **`<p>...</p>`** tags.
+
+Newlines can be added at the end of a line with **`<br />`**. Note that this does not need a closing tag, because the `/` within the angle brackets makes it self-closing.
+
+Generally, you should use **`<p>`** tags for each paragraph and should not use **`<br />`** unless you have a good reason.
+
+### Lists
+
+Numbered lists (e.g., a series of numbered sections) and unnumbered lists (e.g., a list with bullet points) are both represented the same way in the XML template language.
+
+Here are the basic rules:
+
+* The list as a whole should be surrounded with **`<list>...</list>`** tags.
+* Within the `<list>`, each item in the list should be surrounded with **`<item>...</item>`** tags.
+* If a list item has a section number or a bullet point at the start, the section number or bullet should be surrounded with **`<bullet>...</bullet>`** tags.
+* Sub-lists can be nested within an item, to indicate e.g. subsections within an overarching numbered section.
+* Items can also contain **`<p>...</p>`** tags, as needed if an item contains multiple paragraphs.
+
+Example:
+
+```
+1. text 1
+2. text 2
+  * text abc
+  - text def
+```
+
+could be represented as:
+
+```xml
+<list>
+    <item>
+        <bullet>1.</bullet> text 1
+    </item>
+    <item>
+        <bullet>2.</bullet> text 2
+        <list>
+            <item>
+                <bullet>*</bullet> text abc
+            </item>
+            <item>
+                <bullet>-</bullet> text def
+            </item>
+        </list>
+    </item>
+</list>
+```
+
+### Optional text
+
+If a license template should match regardless of whether or not a particular section of text is present, then that text can be surrounded with **`<optional>...</optional>`** tags.
+
+Note that some annotated portions of text (specifically `<titleText>` and `<copyrightText>`) are deemed to be "optional" automatically, so the `<optional>` tag is not needed where those are present.
+
+### Replaceable text
+
+Some licenses contain portions of language that can match to a variety of alternative texts. This is called "replaceable text", and is denoted with the **`<alt>...</alt>`** tag.
+
+The **<alt>** tag includes two attributes:
+* `name`: specifies a unique name for this matching element
+* `match`: specifies a [POSIX extended regular expression (ERE)](http://pubs.opengroup.org/onlinepubs/9699919799/) for what text will match
+
+Note that an example of actually-matching text should be included between the `<alt>` and `</alt>` tags, as the specified example is what will be rendered on the SPDX License List website page.
```

Comment:
We should specify that the <alt match="RE">Some text</alt> "Some Text" will be rendered on the web pages, text files, etc and should be the canonical text, as much as is possible with some licenses....

Reply:
Yup, good call. I'll update to note that too.